<?xml version="1.0"?>

<launch>

 <!-- uwb driver node -->

    <!-- configuration for the network -->
	<rosparam param="/uwb_driver_node0/mobsId"> [200, 201, 202, 203]</rosparam>
	<rosparam param="/uwb_driver_node0/ancsPos"> [4.534, -3.713, 1.945, 2.979, 3.031, 0.6718, -3.518,  3.72, 2.0, -3.011 , -2.985, 0.6213]</rosparam>
	<rosparam param="/uwb_driver_node0/ancsId"> [100, 101, 102, 103]</rosparam>
	<rosparam param="/uwb_driver_node0/albega"> [1.001966856066302, 0.108814649293333, 200]</rosparam> <!-- calibration values for the p4xx -->
	
  <node respawn="true" pkg="uwb_driver" type="uwb_driver_node" name="uwb_driver_node0" output="screen">
            <!-- configuration for the node -->
	<param name="p44QueryRate" type="int" value="20"/>
	<param name="reportRate" type="int" value="20"/>
	<param name="restEnable" type="bool" value="true"/>
	<param name="ignrTimeoutUwbInit" type="bool" value="true"/>
	<param name="publishUwbInfo" type="bool" value="true"/>

            <!-- configuration for the p4xx -->
	<param name="p4xxInterfaceChoice" type="int" value="1"/>
	<param name="p4xxSerialPort" type="string" value="206" />
	<param name="p4xxMode" type="int" value="1" />
	<param name="autoConfigRn" type="bool" value="false" />	

  </node>
  
</launch>

